packages/blade/src/utils/getPlatform/index.ts

```diff
@@ -0,0 +1,2 @@
+export { default } from './getPlatform';
```

Comment:
We named the file `getPlatform` and even using it as `getPlatform()` but the function name is `getPlatformType`

Reply:
nice catch. the intention was `getPlatformType` itself. fixing this